src/test/resources/Observation/create-pa02.json

```diff
@@ -0,0 +1,65 @@
+{
+  "resourceType": "Observation",
+  "id": "e39a18a6-63d7-4117-a38e-1f29be15f6fa",
+  "meta": {
+    "profile": [
+      "https://www.netzwerk-universitaetsmedizin.de/fhir/StructureDefinition/oxygen-partial-pressure"
+    ]
+  },
+  "identifier": [
+    {
+      "type": {
+        "coding": [
+          {
+            "system": "http://terminology.hl7.org/CodeSystem/v2-0203",
+            "code": "OBI"
+          }
+        ]
+      },
+      "system": "https://www.charite.de/fhir/CodeSystem/lab-identifiers",
+      "value": "2703-7_paO2",
+      "assigner": {
+        "reference": "Organization/Charité"
+      }
+    }
+  ],
+  "status": "final",
+  "category": [
+    {
+      "coding": [
+        {
+          "system": "http://loinc.org",
+          "code": "26436-6"
+        },
+        {
+          "system": "http://terminology.hl7.org/CodeSystem/observation-category",
+          "code": "laboratory"
+        },
+        {
+          "system": "http://loinc.org",
+          "code": "18767-4"
+        }
+      ]
+    }
+  ],
+  "code": {
+    "coding": [
+      {
+        "system": "http://loinc.org",
+        "code": "2703-7",
+        "display": "Oxygen [Partial pressure] in Arterial blood"
+      }
+    ],
+    "text": "Oxygen [Partial pressure] in Arterial blood"
+  },
+  "subject": {

```

Comment:
Subject is not correct needs to be fixed

Reply:
Adjusted to 
```
"subject": {
    "identifier": {
      "system": "urn:ietf:rfc:4122",
	  "value": "{{patientId}}"
	}
  }
```